debian8/guide/system/fs-part/installation-storage-partitioning/partition_for_home.rule

```diff
@@ -21,4 +21,4 @@ references:
     disa: 1208
     nist: SC-32
 
-ocil: '<partition-check-macro part="/home " />'
+ocil: '{{{ partition_check(part="/home ") }}}'
```

Comment:
@matejak any reason for the space in all of the partition_checks e.g. ```"/home "``` instead of ```"/home"```

Reply:
It was just backwards compatibility :-)